### DISCUSSION TITLE
Resolve on typing does not work correctly.

### DISCUSSION THREAD

**Discussion by lucasantoniooficial** [rebing/graphql-laravel#1061] [General]
I have a user typing.

I wanted to create a field in this typing for role_id that is in a many-to-many relationship.

But since there is no role_id in the users table, it keeps giving column errors, shouldn't the resolve function be activated for my case? In which I implemented...

**Comment by mfn** [comment]
I think it's best if you share as much code as possible; also I suggest to take a look at the test suite for examples / inspiration.

**Reply by lucasantoniooficial** [reply]
I'm realizing that using $field->getSelect() is causing a lot of trouble.

Removing it, everything works perfectly.
